app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt

```diff
@@ -635,3 +642,10 @@ fun DaxCta.canSendShownPixel(): Boolean {
     val param = onboardingStore.onboardingDialogJourney?.split("-").orEmpty().toMutableList()
     return !(param.isNotEmpty() && param.any { it.split(":").firstOrNull().orEmpty() == ctaPixelParam })
 }
+
+fun String.getStringWithCorrectFinger(position: OmnibarPosition): String {
```

Comment:
this needs a test

Reply:
Added [here](https://github.com/duckduckgo/Android/pull/4942/commits/2dac278adec66ced784a241ca14f1318284fea39).